src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int randNumbersColors = new Random().nextInt(DifferentColors.values().length);
```

Comment:
Random must be a class field
```suggestion
        int randNumbersColors = random().nextInt(Color.values().length);
```